libs/analytics/src/index.ts

```diff
@@ -1,7 +1,17 @@
-export { PixelAnalytics, PixelEvent } from './pixels/PixelAnalytics'
-export { WebVitalsAnalytics } from './webVitals/WebVitalsAnalytics'
+// Core analytics initialization
+export { initGtm } from './gtm/initGtm'
 export { initPixelAnalytics } from './pixels/initPixelAnalytics'
-export { CowAnalyticsGoogle } from './googleAnalytics/CowAnalyticsGoogle'
-export { initCowAnalyticsGoogle } from './googleAnalytics/initGoogleAnalytics'
-export * from './context/CowAnalyticsContext'
-export * from './CowAnalytics'
+export { WebVitalsAnalytics } from './webVitals/WebVitalsAnalytics'
+
+// GTM tracking
+export type { GtmClickEvent, GtmCategory } from './gtm/types'
+export { isValidGtmClickEvent, toGtmEvent } from './gtm/types'
```

Comment:
`isValidGtmClickEvent` is not used outside of the library, no need to export it

Reply:
Addressed